src/layer/x86/convolution_x86.cpp

```diff
@@ -85,7 +86,7 @@ int Convolution_x86::create_pipeline(const Option& opt)
         int num_input = weight_data_size / 9 / num_output;
         // winograd is slow on small channel count
         if(num_input >= 16 && num_output >= 16)
-            use_winograd3x3 = true;
+            use_winograd3x3 = false;
```

Comment:
intentional change ?

Reply:
Yes!
Because the x86 winograd TODO optimizate.